src/OPLSNipals.js

```diff
@@ -21,12 +21,31 @@ export function OPLSNipals(x, y, options = {}) {
 
   let X = Matrix.checkMatrix(x);
```

Comment:
Please change the doc because x and y may be array as well and options default value is {}